node/image-classification-with-huggingface/src/main.js

```diff
@@ -0,0 +1,49 @@
+import { Client, Storage } from 'node-appwrite';
+import { HfInference } from '@huggingface/inference'
+
+export default async ({ req, res, log, error }) => {
+  if (req.method !== 'POST') {
+    return res.send('Method Not Allowed', 405)
+  }
+
+  if (!req.body.image) {
```

Comment:
Should we call it `fileId`? or `imageId`

Reply:
`imageId` feels like it matches better